Source/UI/NewUI/BillingForm/Style Protocol/BillingFormHeaderCellStyle.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import UIKit
+
+public protocol BillingFormHeaderCellStyle {
```

Comment:
@ehab-al-cko I thing which we forgot to document is API spec for our new component work. We can do this now as well so the future tickets will be easier. We can use this PR as a base work and decide on how our public API spec would look like. Please refer existing API specs for Checkout iOS SDK, @harry-brown-cko any please feel free to add.